### PR TITLE
Add profiles

### PR DESCRIPTION
**Pull request by shanecorcoran**
This is my first PR on this, be as hard as you like. This change is to add profiles to dataform to allow for different defaults to be applied. I

### CODE REVIEW

File: cmd/create.go [hunk 1]
```diff
@@ -61,17 +63,18 @@ func createFunc(cmd *cobra.Command, args []string) {
...
-		Engine:             &dbEngine,
-		EngineVersion:      &dbEngineVersion,
-		DBInstanceClass:    &dbInstanceClass,
-		Name:               &name,
-		MultiAZ:            &dbMultiAZ,
-		Port:               &dbPort,
-		StorageAllocatedGB: &dbStorageAllocatedGB,
-	}
+	dbinput := &db.DB{}
+	dbinput.MasterUsername = &dbMasterUsername
```
Comment (pr8kerl) [inline cmd/create.go]: I'll be harsh now.

I don't like this style of instantiation. Prefer the previous. :)

File: pkg/db/db_internal_test.go [hunk 2]
```diff
@@ -53,13 +53,14 @@ func TestCreate(t *testing.T) {
...
-			DBInput := &DB{
-				Name:               &tC.name,
-				MasterUsername:     &tC.username,
-				MasterUserPassword: &tC.password,
-				MultiAZ:            &tC.multiaz,
-				DBInstanceClass:    &tC.dbinstanceclass,
-			}
+			DBInput := &DB{}
+
+			DBInput.Name = &tC.name
```
Comment (pr8kerl) [inline pkg/db/db_internal_test.go]: urgh! My eyes!!

**Comment by gugahoi** [comment]
:fireworks:

File: pkg/db/db_internal_test.go (hunk 2 above)
Comment (shanecorcoran) [reply pkg/db/db_internal_test.go]: See above

File: cmd/create.go (hunk 1 above)
Comment (shanecorcoran) [reply cmd/create.go]: Yep, I agree unfortunately it does not work with composite objects.

**Pull request merged**